documents/knowledge/elements.yaml

```diff
@@ -3266,7 +3267,455 @@
 
 # DIGRAPHS
 
-- element: "bø"
+- element: "∆c"
+  name: Cosine
+  arity: 1
+  description: Get the cosine of an angle in radians
+  overloads:
+    num: math.cos(a)
+  vectorise: true
+  tests:
+    - "[3.14159265358979] : -1"
+    - "[0] : 1"
+    - "[6.283185307] : 1"
+
+- element: "∆C"
+  name: Arc Cosine
+  arity: 1
+  description: Get the arccosine of an angle in radians
+  overloads:
+    num: math.arrcos(a)
+  vectorise: true
+  tests:
+    - "[-1] : 3.14159265358979"
+    - "[1] : 0"
+
+- element: "∆q"
+  name: Quadratic Solver
+  arity: 2
+  description: Solve a quadratic equation of the form ax^2 + bx = 0
+  overloads:
+    num-num: x such that ax^2 + bx = 0
+    num-str: solve a such that a = b
+    str-num: solve b such that b = a
+    str-str: solve equation a = b for x
+  vectorise: true
+  tests:
+    - "[1, 2] : [-2, 0]"
+    - "[1, -2] : [0, 2]"
+    - "[69, 420] : [-140/23, 0.0]"
+
+- element: "∆Q"
+  name: General Quadratic Solver
```

Comment:
I don't understand why we can't merge this and ∆q into one for a truly general quadratic solver.

Reply:
I think the point is this is a shortcut for if you know that `a` is definitely going to be 1, and `∆q` is for if you know `c` is definitely `0`, so it's a little bit shorter.